enterprise/core-edge/src/main/java/org/neo4j/coreedge/raft/log/segmented/SegmentFile.java

```diff
@@ -262,4 +262,17 @@ public long size()
     {
         return fileSystem.getFileSize( file );
     }
+
+    @Override
+    public void close() throws DisposedException
+    {
+        if ( closed )
+        {
+            throw new RuntimeException( CLOSED_ERROR_MESSAGE );
```

Comment:
I don't think this is the right place to log, indeed we do not log `DisposedException` either.